src/ocamlorg_frontend/pages/package_documentation.eml

```diff
@@ -41,60 +44,40 @@ let path_page = match str_path |> String.concat "/" with
 Package_layout.render
-~title
-~description:(Printf.sprintf "%s %s: %s" package.name package.version package.description)
+~is_latest_url
+~title:(if is_latest_url then (Printf.sprintf "%s %s · OCaml Package" package.name "latest") else (Printf.sprintf "%s %s · OCaml Package" package.name package.version))
+~description:(if is_latest_url then (Printf.sprintf "%s %s: %s" package.name "latest" package.description) else (Printf.sprintf "%s %s: %s" package.name package.version package.description))
 ~package
 ~path
-~canonical:(Url.package_doc package.name package.version ?page:path_page)
+~canonical:(Url.package_doc package.name ?version ?page:path_page)
 ~styles:["/css/main.css"; "/css/doc.css"] @@
-<div x-data="{ open: false, sidebar: window.innerWidth > 1024 && true, showOnMobile: false}" @resize.window="sidebar = window.innerWidth > 1024" x-on:close-sidebar="sidebar=window.innerWidth > 1024 && true">
+<div x-data="{ open: false, sidebar: window.innerWidth > 1024 && true, showOnMobile: false}" @resize.window="sidebar = window.innerWidth > 1024">
   <button :title='(sidebar? "close" : "open")+" sidebar"' aria-pressed="false" :aria-pressed="sidebar" class="flex items-center bg-primary-600 p-3 z-30 rounded-full text-white shadow-md bottom-20 fixed lg:hidden right-10"
     x-on:click="sidebar = ! sidebar">
     <%s! Icons.sidebar_menu "h-8 w-8" %>
     <span class="hidden md:flex font-semibold px-2">side menu</span>
   </button>
-  <div class="fixed z-10 inset-0 bg-background-dark-blue/20 backdrop-blur-sm lg:hidden"
-    :class='sidebar ? "" : "hidden"' aria-hidden="true" x-on:click="sidebar = ! sidebar"></div>
 
   <div class="flex flex-col lg:flex-row md:gap-12">
     <div
       class="p-10 z-20 bg-white flex-col fixed h-screen overflow-auto lg:pr-0 lg:flex left-0 top-0 lg:sticky w-72 lg:p-0 lg:pt-6"
       x-show="sidebar" x-transition:enter="transition duration-200 ease-out"
       x-transition:enter-start="-translate-x-full" x-transition:leave="transition duration-100 ease-in"
       x-transition:leave-end="-translate-x-full">
-      <%s! sidebar ~str_path ~toc ~maptoc package %>
+      <%s! sidebar ~str_path ~toc ~maptoc ~is_latest_url package %>
     </div>
-    <div class="flex-1 z-0 z- min-w-0 lg:max-w-3xl lg:pt-6" id="htmx-content">
+    <div class="flex-1 z-0 z- w-full lg:max-w-3xl relative lg:pt-6">
       <div class="odoc prose prose-orange">
         <%s! content %>
       </div>
     </div>
-    <div class="hidden xl:flex top-0 sticky h-screen flex-col w-60 overflow-auto right-sidebar" id="htmx-right-sidebar">
-      <%s! Toc.render toc %>
+    <div class="hidden xl:flex top-0 sticky h-screen">
+      <div class="flex-col w-60">
+        <div class="h-screen overflow-scroll right-sidebar pt-6">
+          <div class="font-semibold text-gray-500 text-sm mb-4">ON THIS PAGE</div>
+          <%s! Toc.render toc %>
+        </div>
+      </div>
```

Comment:
Keep this the same as on the main branch. Your patch shouldn't make changes here.